app/test.py

```diff
@@ -1,40 +1,56 @@
 import spacy
+import os
 import torch
-from core.nlu.spacy import SpacyModel
-from core.summarizer.textrank import TextRank
-from core.language.gpt2 import GPT2
+from song_story import SpacyModel
+from song_story import TextRank
```

Comment:
shift tests file to test folder

Reply:
Removed the test.py, it was actually being used to demo the functions while developing, should have used different name